### ISSUE TITLE
Chatterino 2 client zoomed in along with tab titles/can't click users and buttons.

### ISSUE THREAD

**Issue by blueNcream** [Chatterino/chatterino2#1363]
**Describe the bug**
The entire chat and tab titles are zoomed-in/superimposed/blown up. Could not press CTRL+ mouse wheel scroll down to change size on version 2.1.4. The font scaling was fixed on 2.1.5 but chat is still large/zoomed-in. Scaling down ruins the image/view of the client.

**To Reproduce**
One day the client was "Not Responding" from no action. Closed it and turned off PC.
The next day Chatterino 2 opens in zoomed in view.

**Expected behavior**
Normal view.

**Screenshots**
https://imgur.com/a/9Irc06t

**Chatterino version**
Chatterino 2.1.5 (commit c6e1a1de2)

**Additional context**
Cog can't be clicked if maximized and the minimize, maximize and close buttons do not work/doesn't click. The user names can't be clicked. This is from a work PC. Personal laptop does not have this issue.

**Comment by pajlada** [comment]
Sounds like a hi-dpi issue, could you share the resolution, scaling% of your work PC?

**Comment by blueNcream** [comment]
> Sounds like a high dpi issue, could you share the resolution, scaling% of your work PC?

Thanks for noticing that. I tried within windows to solve this and although only the tab title was fixed, the chat remains blurry. Users and buttons are still not able to be clicked.

So with that info, the problem started when chatterino2 crashed (not responding) on monitor 2. It was maximized there as always.

work PC setup:
Monitor 1 3840 x 2160 (Recommended) landscape 150% scaling (Recommended) DP cable
Monitor 2 1050 x 1680 (Recommended) portrait 100% scaling (Recommended) HDMI cable
win10pro i7 GTX780
